guides/common/modules/proc_enabling-the-satellite-tools-repository.adoc

```diff
@@ -15,8 +15,10 @@ endif::[]
 . Use the Search field to enter the following repository name: *{project-client-name} (for RHEL 8) (RPMs)*.
 . In the Available Repositories pane, click on *{project-client-name} (for RHEL 8) (RPMs)* to expand the repository set.
 +
-If the *{project-client-name}* items are not visible, it may be because they are not included in the Red{nbsp}Hat Subscription Manifest obtained from the Customer Portal.
-To correct that, log in to the Customer Portal, add these repositories, download the Red{nbsp}Hat Subscription Manifest and import it into {Project}.
```

Comment:
What was wrong with this? The original first sentence was clearer.

Reply:
I just reworded it.

Reply:
Because the sentence `it may be...` wasn't straight and clear. However, I am open to restoring it.

Reply:
I'd keep the two original lines, because we're just adding a link that gives more details about these two lines.

Reply:
Addressed :+1: